### ISSUE TITLE
mp_gga_relax_job pseudopotential parsing bug

### ISSUE THREAD

**Issue by zulissimeta** [Quantum-Accelerators/quacc#2457]
### Details about the quacc environment

- **quacc version**: 0.11.3
- **Python version**: 3.11

### What is the issue?

The pseudopotential symbols are not getting split correctly in the VASP set converter. 

```
  File "../lib/python3.11/site-packages/quacc/calculators/vasp/params.py", line 471, in _convert
    potcar_setups[k] = v.split(k)[-1]
                       ^^^^^^^^^^
ValueError: empty separator
```

This appears to be because the list of pseudopotential symbols (which is separated by `\n` is not getting split into a list, so each character is treated separately. 

### How can we easily reproduce the issue?

```python
from ase.build import bulk
from quacc import flow
from quacc.recipes.vasp.mp import mp_gga_relax_job


@flow
def test():
    return mp_gga_relax_job(bulk("Cu"))


test()
```

**Comment by Andrew-S-Rosen** [comment]
Thank you for the report, @zulissimeta! This looks like a new bug due to the release of atomate2 0.0.15. We aren't testing it on CI yet because of a dependency conflict (https://github.com/materialsproject/atomate2/pull/970). Happy to accept a fix.

**Comment by Andrew-S-Rosen** [comment]
Fixed by @chiang-yuan in #2463!
